utils.go

```diff
@@ -0,0 +1,57 @@
+package multilogger
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// FormatDuration returns a string to represent the duration.
+func FormatDuration(duration time.Duration) string {
+	const day = 24 * time.Hour
+	const week = 7 * day
```

Comment:
If I'm reading a duration, weeks and months are actually less clear for me. Might just be me.
I'd much rather have 105 days than 3 months 2 weeks 1 day

Reply:
It really depends, what is 178 days? In either case, I hope our program won't run for so long.